src/TemplatePassthroughManager.js

```diff
@@ -45,15 +44,21 @@ class TemplatePassthroughManager {
     let target = this.config.passthroughCopies || {};
     debug("`passthroughFileCopy` config paths: %o", target);
     for (let path in target) {
-      paths.push(TemplatePath.addLeadingDotSlash(path));
+      const inputPath = TemplatePath.addLeadingDotSlash(path);
+      const outputPath =
+        typeof target[path] === "string"
```

Comment:
If string, assume an explicit output path. Else assume boolean and match input path